applicationset/webhook/webhook_test.go

```diff
@@ -199,6 +199,7 @@ func TestWebhookHandler(t *testing.T) {
 		t.Run(test.desc, func(t *testing.T) {
 			fc := fake.NewClientBuilder().WithScheme(scheme).WithObjects(
 				fakeAppWithGitGenerator("git-github", namespace, "https://github.com/org/repo"),
+				fakeAppWithGitGenerator("git-github-copy", namespace, "https://github.com/org/repo-copy"),
```

Comment:
Do we need to do extra checks below to make sure the webhook is not triggered wrongly, or are these checks already present?

Reply:
Checks are already existing :+1: 
Here we have the list of expected appset to be refreshed : https://github.com/argoproj/argo-cd/blob/master/applicationset/webhook/webhook_test.go#L66
and here we have a condition to detect if an non expected appset has been refreshed or not : https://github.com/argoproj/argo-cd/blob/master/applicationset/webhook/webhook_test.go#L246